vault/changelog.release.2022-05-17.md

```diff
@@ -0,0 +1,84 @@
+---
+id: o674526ctxg6rt8bc144rdz
+title: 0.95
+desc: ''
+updated: 1652806373844
+created: 1652755065976
+---
+
+Dendron 0.95 has sprouted  🌱
+
+**Local Configuration:** We are releasing an early build of Config Overrides, which allows a local configuration to override values in the default configuration. This lets you use your own settings and vaults within a shared team workspace.
+
+- More information here about [[local configurations|dendron://dendron.dendron-site/dendron.topic.local-config-override]]
+
+**Custom Themes:** You can now customize the style of your sites published with Dendron. Step right up, bring your own custom CSS theme, and be amazed with how good your knowledge can look!
+
+- Learn more about [[custom themes|dendron://dendron.dendron-site/dendron.topic.theme.publish.md]]
+
+**Initialization Speed-Up:** But wait, there's more: less waiting! This release drastically improves workspace initialization speed (10x faster for large workspaces 🚀 ).
+
+## Highlights
+- feat(chore): germ stage implementation of config overrides 
```

Comment:
i realized i wrote this but its a terrible description. change to "preview support for local configuration overrides"